infectiousdiseases/patient_lists.py

```diff
@@ -0,0 +1,62 @@
+from opal.core.patient_lists import PatientList, TaggedPatientList
```

Comment:
I think we've lost the immune_inpatients, immune_liaison (+ immune parent?) and the tropical lists somehow